src/elements/README.md

```diff
@@ -4,8 +4,12 @@ The Box UI Elements are designed to be agnostic of the authentication type. They
 _Note: When you are required to pass an access token to a Box Element Component, keep in mind that you can pass the access token as a string, or as a function. If you pass a function, it should return a string. Whenever the Box Element Component needs to make an API call to Box, it will use the accessToken you supplied, or invoke the access token function to get the access token._
 
 ## Internationalization (i18n)
-The UI Elements use [react-intl](https://github.com/formatjs/react-intl) for internationalization. In order for the UI Elements to render properly, they need to be wrapped in an i18n context via the [IntlProvider](https://github.com/formatjs/react-intl/blob/master/docs/Getting-Started.md#creating-an-i18n-context) and given a locale and translated messages to use. Each of the UI Element components optionally take in a `language` property and a `messages` property which is then delegated to our internal [IntlProvider](src/components/Internationalize.js). If either of these properties are not passed in, we do not use our internal `IntlProvider` and it is assumed that the parent react app (the react app that is importing in the UI Elements) has its own `IntlProvider`.
+The Box UI Elements utilize an internal [IntlProvider](https://github.com/formatjs/formatjs/blob/main/website/docs/react-intl/components.md#intlprovider) system powered [react-intl](https://formatjs.io/docs/react-intl/). To make use of our internal `IntlProvider`, both `language` and `messages` properties must be provided to the UI Element components.
+
+If you choose not to use the internal `IntlProvider`, you are expected to provide your own `IntlProvider`. In this case, you still need to provide the `language` and `messages` properties to your implementation of `IntlProvider`. This is because the UI Elements are designed to work within an `IntlProvider`, and they rely on these properties to function correctly.
```

Comment:
> If you choose not to use the internal `IntlProvider`, you are expected to provide your own `IntlProvider`.

I think this wording might actually make it more confusing about what it means to "provide" your own IntlProvider. for example, does the developer need to provide it to the element like a prop? does each element needs it's own provider if you don't use the internal one?

the wording from the original seemed more clear to me because it doesn't expect that each element needs their own individual provider:
```
it is assumed that the parent react app (the react app that is importing in the UI Elements) has its own `IntlProvider`.
```

Is there anything specific from the original paragraph that's not clear? i think it flows a little better